### PR TITLE
feat: video actions for short-video-service

### PR DESCRIPTION
**Pull request by BaiZe1998**
## Link to an issue #
短视频上传功能的 CoreService 部分
#10 

## Checklist

- [ ] Link to an issue if it really related.
- [ ] At least describe what this PR does.
- [ ] Use `rebase` to confirm that current branch doesn't conflict with main branch.
- [ ] Unit tests. At least do not reduce the unit test coverage.
- [ ] Checked and updated `guidelines.md` which used to describe how to build, deploy and use DouTok.

### CODE REVIEW

File: backend/shortVideoCoreService/internal/data/model/video.go [hunk 1]
```diff
@@ -1 +1,22 @@
...
+const TableNameVideo = "video"
+
+type Video struct {
+	ID           int64     `gorm:"column:id;type:bigint(20);primaryKey"`
+	UserID       int64     `gorm:"column:user_id;type:bigint(20);index"`
+	Title        string    `gorm:"column:title;type:varchar(255)"`
+	Description  string    `gorm:"column:description;type:varchar(255)"`
+	VideoURL     string    `gorm:"column:video_url;type:varchar(255)"`
+	CoverURL     string    `gorm:"column:cover_url;type:varchar(255)"`
+	LikeCount    int64     `gorm:"column:like_count;type:bigint(20)"`
```
Comment (TremblingV5) [inline backend/shortVideoCoreService/internal/data/model/video.go:14]: 后面喜欢数这些东西要存在这张表吗

File: backend/shortVideoCoreService/internal/domain/videodomain/video.go [hunk 2]
```diff
@@ -1 +1,141 @@
...
+
+	resp, err := uc.videoRepo.GetVideoFeed(ctx, &domain_dto.GetVideoFeedRequest{
+		UserId:     request.UserId,
+		LatestTime: latestTime,
+		Num:        request.FeedNum,
+	})
+	if err != nil {
+		return nil, err
+	}
+	user, err := uc.userRepo.FindByID(ctx, request.UserId)
```
Comment (TremblingV5) [inline backend/shortVideoCoreService/internal/domain/videodomain/video.go:61]: 这里是不是需要获取视频的作者，而不是request里面的user id

File: backend/shortVideoCoreService/internal/pkg/auth/jwt.go [hunk 3]
```diff
@@ -1,20 +1,49 @@
...
+		"exp":     time.Now().UTC().Add(time.Hour * hours).Unix(),
 	})
 	signedToken, err := token.SignedString([]byte(config.Auth.Secret))
 	if err != nil {
 		return "", err
 	}
 	return signedToken, nil
 }
+
+func GetLoginUser(ctx context.Context) (int64, error) {
```
Comment (TremblingV5) [inline backend/shortVideoCoreService/internal/pkg/auth/jwt.go:25]: 这个方法我们后续是不是可以移动到api服务里，内部服务之间感觉没必要去鉴权了

Comment (BaiZe1998) [reply backend/shortVideoCoreService/internal/pkg/auth/jwt.go:25]: 是的

Comment (BaiZe1998) [reply backend/shortVideoCoreService/internal/pkg/auth/jwt.go:25]: 这个是我本地用 http 调试的 token

File: backend/shortVideoCoreService/internal/data/model/video.go (hunk 1 above)
Comment (BaiZe1998) [reply backend/shortVideoCoreService/internal/data/model/video.go:14]: 后续可以拆出去

File: backend/shortVideoCoreService/internal/domain/videodomain/video.go (hunk 2 above)
Comment (BaiZe1998) [reply backend/shortVideoCoreService/internal/domain/videodomain/video.go]: yes，已修正

**Review by TremblingV5 — APPROVED** [review]

**Pull request merged**